github_handler.py

```
@@ -0,0 +1,358 @@
+            owner = match.group(1)
+            repo = match.group(2)
+            
+            repo = repo.rstrip('.git')
+            
+            return {
+                'owner': owner,
+                'repo': repo
+            }
+        
+        logger.debug(f"No repo info found in URL: {url}")
+        return None
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)